filters/ThirdParty/filter_256_Scam_Blocklist/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "filterId": 256,
+  "name": "Scam Blocklist",
```

Comment:
We should mention the author here.

Let's do the same as in HostlistsRegistry:
```
Name: Scam Blocklist by DurableNapkin
Description: List for blocking untrustworthy websites
```